com.sublimehq.SublimeText.yaml

```diff
@@ -0,0 +1,88 @@
+app-id: com.sublimehq.SublimeText
+command: subl
+
+runtime: org.freedesktop.Sdk
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --talk-name=org.gnome.SettingsDaemon
+  - --filesystem=host
```

Comment:
that's a broad permission

Reply:
It's what the Sublime Text 3 Flatpak has, I am open to suggestions.

Reply:
I would say `xdg-documents` but I'm sure not everyone would agree. Does Sublime Text use portals ? it should.

Reply:
It is a "ide", xdg-documents will make it useless for everyone

Reply:
I use Sublime Text to edit everything, including stuff that requires `sudo`, if we reduce that functionality, I imagine no one will be very happy about it.

Reply:
I'm not sure whether it uses portals. Sublime Text 3 doesn't, in any case, and is unlikely to be updated to do so now that Sublime Text 4 exists. Could we test whether it does use portals and then use that information to make a decision?

Reply:
I don't know how to test that.

Reply:
It must use portals, because I removed the permission and I am still able to open, edit, and save files.

Reply:
The only weird thing is that it shows the wrong path in the title bar now, which I would find unbelievably annoying in daily use.

Reply:
More details: https://github.com/flathub/flathub/pull/2855#issuecomment-1043888847